src/components/Spinner/tests/Spinner.test.tsx

```diff
@@ -15,24 +15,24 @@ describe('<Spinner />', () => {
   describe('size', () => {
     it('renders a large spinner by default', () => {
       const spinner = shallowWithAppProvider(<Spinner />);
-      expect(spinner.find('svg').prop('viewBox')).toBe('0 0 44 44');
+      expect(spinner.find('svg').hasClass('sizeLarge')).toBeTruthy();
```

Comment:
If `hasClass` is a valid thing to test, why don't we use it elsewhere, I wonder?